logbook-parent/pom.xml

```diff
@@ -44,7 +44,6 @@
         <spring-security4.version>4.2.20.RELEASE</spring-security4.version>
 
         <spring5.version>5.3.25</spring5.version>
-        <spring-boot2.version>2.6.4</spring-boot2.version>
```

Comment:
If this is meant to stop supporting Spring Boot 2, wouldn't it make sense to also drop support for Spring Boot 1 at this point (see above).

Reply:
I'm about to bring the property back as it's still used by spring4 profile (that was a miss). But I'd suggest dropping the support of Spring Boot 1 overall.